FluidNC/src/Main.cpp

```diff
@@ -197,14 +197,55 @@ void loop() {
 
 void WEAK_LINK machine_init() {}
 
-#    if 0
```

Comment:
This needs to be factored somehow.  We do not favor platform-dependent changes to core code guarded by ifdefs.  Allowing that would make the code turn into a mess over time.

Reply:
Sounds reasonable, will move the main function for FluidConsole to separate file in X86TestSupport folder